lms/envs/common.py

```diff
@@ -3294,6 +3294,11 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
     'DEFAULT_RENDERER_CLASSES': (
         'rest_framework.renderers.JSONRenderer',
     ),
+    'DEFAULT_AUTHENTICATION_CLASSES': [
+        'edx_rest_framework_extensions.auth.jwt.authentication.JwtAuthentication',
```

Comment:
My concern, originally documented in [this ARCHBOM-107 comment](https://openedx.atlassian.net/browse/ARCHBOM-107?focusedCommentId=515518), was that this may add risk until https://github.com/openedx/edx-drf-extensions/pull/197 (use forgiving JWTs) lands, but now I'm not so sure.  Can you check my thinking on this?

1. Most endpoints used by MFE already added JwtAuthentication, so those would be unaffected.
2. Rare endpoints, like some bookmarks API, are used by MFEs and never got JwtAuthentication added. Although it is possible that JwtAuthentication could fail and SessionAuthentication would never be tried, this would be no different all the other endpoints in case 1 above, so this _should_ be a net win.
3. Any endpoint that is not used by an MFE, would not be using the `HTTP_USE_JWT_COOKIE` (which is only sent by the MFE), so the JWT cookie would not be available and this should not have any net affect.

So, I now think this is safe. Agreed?

Reply:
I'm marking this as resolved. I think it is safe for me to deploy this before enabling forgiving JWTs.

Reply:
@feanil: I may be changing my tune around this. It seems that there are plenty of MFE endpoints that are using the default classes, which don't have `JwtAuthentication`, and end up using `SessionAuthentication`. With this change, we may add additional failures that don't allow `SessionAuthentication` to recover. So, I'm now thinking I'd like to get forgiving JWTs rolled out first to hopefully reduce the number of errors seen by users.